static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -39,6 +41,11 @@ define([
         setProgressTracker(atomId);
         tracking.track('play', getTrackingId(atomId));
 
+        var mainContainer = document.querySelectorAll('.immersive-main-media .youtube-media-atom');
```

Comment:
I think you could just do `document.querySelector` then `if (mainContainer)`
Also, I think this is better named `immersiveMainContainer`